eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/tcp/impl/eventmeshmessage/EventMeshMessageTCPSubClient.java

```diff
@@ -126,7 +126,7 @@ public void close() {
         try {
             super.close();
         } catch (Exception ex) {
-            ex.printStackTrace();
+            log.warn("exception occurred when close.", ex);
```

Comment:
I think that's better to use log.error method

Reply:
log is generated by lombok， already change warn to error